qiskit/qpy/binary_io/circuits.py

```diff
@@ -587,7 +626,7 @@ def _write_instruction(file_obj, instruction, custom_operations, index_map, use_
             not hasattr(library, gate_class_name)
             and not hasattr(circuit_mod, gate_class_name)
```

Comment:
Do we not export `AnnotatedOperation` from `qiskit.circuit`? I feel like it should be there as it is a public interface. If  we are wouldn't this line pick it up as an attribute of the circuit module?

Reply:
Indeed, that was an omission on my part. [77d044d](https://github.com/Qiskit/qiskit/pull/11505/commits/77d044da2e15cdd1f89ce250bfe00c6a7cce629d) exports this together with the modifiers. This also removes the need to check the name in the `if` above.